images/kubekins-test/Dockerfile-1.9

```diff
@@ -1,54 +0,0 @@
-# Copyright 2017 The Kubernetes Authors.
```

Comment:
(I don't see a 1.13 Dockerfile?)

Reply:
It already existed before this PR: https://github.com/kubernetes/test-infra/blob/master/images/kubekins-test/Dockerfile-1.13